core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -84,7 +83,10 @@ class ControllerChannelManager(controllerContext: ControllerContext,
   }
 
   def initBrokerResponseSensors(): Unit = {
-    Array(ApiKeys.STOP_REPLICA, ApiKeys.LEADER_AND_ISR, ApiKeys.UPDATE_METADATA, ApiKeys.LI_COMBINED_CONTROL).foreach { k: ApiKeys =>
+    Array(
+      ApiKeys.STOP_REPLICA, ApiKeys.LEADER_AND_ISR, ApiKeys.UPDATE_METADATA,
```

Comment:
nit: split each element to a single line so that future changes are easier to review.

Reply:
Done